insights/parsers/systemd/config.py

```diff
@@ -5,10 +5,11 @@
 ``config``.
 """
 
-from insights.contrib.ConfigParser import RawConfigParser as cp
```

Comment:
Please revert to from "insights.contrib.ConfigParser import RawConfigParser as cp"